domains/Discussions/routes.php

```diff
@@ -39,3 +40,8 @@
     'as' => 'questions.view',
     'uses' => QuestionsViewController::class,
 ]);
+
+Route::get('/questions/{id}/answers', [
```

Comment:
Can we keep the route segment related to the question id with the same name as declared in other routes (`questionId`)?